### PR TITLE
Add local information pages

### PR DESCRIPTION
**Pull request by pabloferz**
This adds pages for

 - Venue showing the maps for getting around
 - Food Options (#390)
 - Social Events (#391)

### CODE REVIEW

**Comment by pabloferz** [comment]
@bvdmitri, thanks for the heads up! (and for using the sub-menus structure I have here). I will update the PR accordingly.

**Comment by Wimmerer** [comment]
lgtm

File: 2023/food.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Food Options
+
+The following food options will be available during JuliaCon:
```
Comment (ranjanan) [inline 2023/food.md:3]: will be available as foodtrucks during JuliaCon. They will be parked right outside the venue next to the Stata amphitheater.

File: 2023/food.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+### Wednesday and Friday
+
+ - Gourmet Kreyol
```
Comment (ranjanan) [inline 2023/food.md:7]: https://www.gourmetkreyol.com

File: 2023/food.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+ - Chicken n Rice Guys
+
+### Thursday
+
+ - Bibimbox
```
Comment (ranjanan) [inline 2023/food.md:12]: https://www.bibimboxtruck.com

File: 2023/food.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+ - Tandoor & Curry on Wheels
```
Comment (ranjanan) [inline 2023/food.md:13]: https://tandoorandcurryonwheels.com

**Pull request merged**